src/promptflow/promptflow/_sdk/operations/_tool_operations.py

```diff
@@ -80,3 +81,9 @@ def _parse_tool_from_function(f, initialize_inputs=None):
             function=f.__name__,
             module=f.__module__,
         )
+
+    @staticmethod
+    def list():
```

Comment:
add docstring for this fucntion, esp. doc the return data obj

Reply:
Add the docstring to list func